lib/active_force/active_query.rb

```diff
@@ -94,14 +94,17 @@ def find_by!(conditions)
 
     def includes(*relations)
       relations.each do |relation|
-        association = sobject.associations[relation]
-        fields Association::EagerLoadProjectionBuilder.build association
-        # downcase the key and downcase when we do the comparison so we don't do any more crazy string manipulation
-        association_mapping[association.sfdc_association_field.downcase] = association.relation_name
+        case relation
+        when Symbol
+          association = sobject.associations[relation]
+          build_includes(association)
+        when Hash
+          build_hash_includes(relation)
+        end
       end
       self
     end
-
+    
```

Comment:
Nitpick: trailing whitespace.  I saw a few more of these when I pulled this into my editor.  Could you check for those?  I know we inherited a pretty messy codebase in terms of formatting, but we don't want to make it worse.

Reply:
removed in latest commit